src/main/asciidoc/wombat-spec.adoc

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) 2017-2019 Contributors to the Eclipse Foundation
+//
+
+= Jakarta Wombat
+:authors: Jakarta Wombat Team, https://projects.eclipse.org/projects/ee4j.wombat
+:email: https://dev.eclipse.org/mailman/listinfo/wombat-dev
+:version-label!:
+:sectanchors:
+:doctype: book
+:license: Eclipse Foundation Specification License v1.0
+:source-highlighter: coderay
+:toc: left
+:toclevels: 4
+:sectnumlevels: 4
+ifdef::backend-pdf[]
```

Comment:
Something in here is turning on :sectnums: before the license (which shouldn't have section numbers), but only for the pdf version.

Reply:
Figured it out.  I can turn off the section numbering for the efsl and turn it back on for the content.